### PR TITLE
Fix last half year bug

### PR DESCRIPTION
**Pull request by rubio41**

### CODE REVIEW

File: Microsoft.Recognizers.Text.DateTime.Tests/English/TestDurationExtractor.cs [hunk 1]
```diff
@@ -44,6 +44,9 @@ public void TestDurationExtract()
 
             BasicTest("I'll leave for an hour", 15, 7);
             BasicTest("I'll leave for a year", 15, 6);
+
+            BasicTest("half year", 0, 9);
+            BasicTest("half an year", 0, 12);
```
Comment (yiylin) [inline Microsoft.Recognizers.Text.DateTime.Tests/English/TestDurationExtractor.cs]: Can you add the Chinese unit test and add TODO in the Spanish unit test?

File: Microsoft.Recognizers.Text.DateTime.Tests/English/TestDurationParser.cs [hunk 2]
```diff
@@ -46,6 +46,10 @@ public void TestDurationParse()
             BasicTest("I'll leave for all year", 31536000, "P1Y");
 
             BasicTest("I'll leave for an hour", 3600, "PT1H");
+
+            BasicTest("half year", 15768000, "P0.5Y");
+            BasicTest("half an year", 15768000, "P0.5Y");
+
```
Comment (yiylin) [inline Microsoft.Recognizers.Text.DateTime.Tests/English/TestDurationParser.cs]: Can you add the Chinese unit test and add TODO in the Spanish unit test?

Comment (rubio41) [inline Microsoft.Recognizers.Text.DateTime.Tests/English/TestDurationParser.cs]: the Chinese extractor and parser are not following the base class. It needs support for number extractor for "半". I will file a bug in VSO.

Comment (yiylin) [inline Microsoft.Recognizers.Text.DateTime.Tests/English/TestDurationParser.cs]: ok, Then I will merge it firstly.

**Pull request merged**